tests/test_data_hub.py

```diff
@@ -375,29 +405,7 @@ def test_download_database(config):  # noqa: F811
         db_id,
         include_files=False,
         client=config["client"],
-    )
-
-    file_name = db_id + ".csv"
-    assert os.path.exists(file_name), f"Expected to find a CSV file called {file_name}"
-    os.remove(file_name)
-
-
-def test_download(config):  # noqa: F811
-    db_id = config["databases"][0]
-    api.download(
-        db_id,
-        destination=os.getcwd(),
-        client=config["client"],
-    )
-
-    file_name = db_id + ".csv"
-    assert os.path.exists(file_name), f"Expected to find a CSV file called {file_name}"
-    os.remove(file_name)
-
-    api.download(
-        f"{PREFIXES.DO}{db_id}",
-        destination=os.getcwd(),
-        client=config["client"],
```

Comment:
this is essentially a duplicated test and introduces a race condition when run in parallel (as is the case when we run against a live instance). so removing